source/stylesheets/base/_layout.scss

```diff
@@ -24,8 +24,11 @@ body {
 }
 
 .container {
-  margin: 0 auto;
-  max-width: 1000px;
+  @include grid-media($large-screen-up) {
+    margin: 0 auto;
+    max-width: 1000px;
+  }
+  max-width: 100%;
```

Comment:
I usually order the declaration first. My logic is that since the media query is cancelling out the default state it should come afterwards. So it would look like this:

```
.container {
  max-width: 100%;

  @include grid-media($large-screen-up) {
    margin: 0 auto;
    max-width: 1000px;  
  }
}
```
This is the rare case where a mixin doesn't live atop the declaration block.